backend/src/routes/api/storage-class/index.ts

```diff
@@ -0,0 +1,26 @@
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { StorageClassConfig, KubeFastifyInstance } from '../../../types';
+import { secureAdminRoute } from '../../../utils/route-security';
+import { updateStorageClassMetadata } from './utils';
+
+export default async (fastify: KubeFastifyInstance): Promise<void> => {
```

Comment:
Why not name this function? This is more of a specialized action for a file called "index" in a folder called "storage-class"

Reply:
- the api works by finding the default export for each folder in `/api`, so there is no need to name the function then export it as default from another file
- this is also the pattern we use in all the other routes